news/2024/2024-01-26-mwc-7k-ro24-summary.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2024: Round of 24 Recap"
+date: 2024-01-26 15:00:00 +0000
+---
+
+The osu!mania 7K World Cup begins! Stay updated as participants start revving up their engines to maintain their momentum along the remainder of the tournament.
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)
+
+Continuing on from the osu! World Cup recaps, you can once more expect a summary of the latest and greatest of MWC, made possible by community members just like you! This year, **for the first time ever**, the tournament kicked off with a Round of 24! Due to increased interest from various countries that have both applied and qualified, we have been blessed with an extra round and exciting matches straight away!
```

Comment:
would be nice to link to an owc recap here
```suggestion
Continuing on from the osu! World Cup [recaps](https://osu.ppy.sh/home/news/2023-11-28-osu-world-cup-2023-finals-recap), you can once more expect a summary of the latest and greatest of MWC, made possible by community members just like you! This year, **for the first time ever**, the tournament kicked off with a Round of 24! Due to increased interest from various countries that have both applied and qualified, we have been blessed with an extra round and exciting matches straight away!
```

Reply:
Can be done. Any reason to choose Finals over the first recap or the final one? Not sure if it was chosen for anything in particular. Otherwise, any hyperlink works really.

Reply:
used the most recent recap.

Reply:
It is what it is. Using the most recent, users can find the others with relative ease.